Can't run `./gradlew projects`

For example:
```
FAILURE: Build failed with an exception.

* Where:
Build file '/Volumes/VAULTROOM/gen-os/stytch-android/b2bExampleApp/build.gradle' line: 33

* What went wrong:
A problem occurred evaluating project ':b2bExampleApp'.
> Could not get unknown property 'STYTCH_B2B_PUBLIC_TOKEN' for BuildType$AgpDecorated_Decorated{name=debug, debuggable=true, testCoverageEnabled=false, jniDebuggable=false, pseudoLocalesEnabled=false, renderscriptDebuggable=false, renderscriptOptimLevel=3, minifyEnabled=false, zipAlignEnabled=true, signingConfig=SigningConfig$AgpDecorated_Decorated{name=debug, storeFile=/Users/sam/.android/debug.keystore, storePassword=android, keyAlias=AndroidDebugKey, keyPassword=android, storeType=pkcs12, v1SigningEnabled=true, v2SigningEnabled=true, enableV1Signing=null, enableV2Signing=null, enableV3Signing=null, enableV4Signing=null}, embedMicroApp=false, mBuildConfigFields={}, mResValues={}, mProguardFiles=[], mConsumerProguardFiles=[], mManifestPlaceholders={}} of type com.android.build.gradle.internal.dsl.BuildType$AgpDecorated.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.7/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
```

But we are trying to test B2C features, and we do not have a B2B application.

--- a/README.md
+++ b/README.md
@@ -225,14 +225,16 @@ With the above in place your app should be ready to accept deeplinks
 
 ### Example app
 
-The above code is used in practice in the Example Apps, which can be run and used to test out various flows of the SDK.
-In order to run the application, you have to define some gradle properties in your global or local `gradle.properties`.
+The above code is used in practice in the included Example Apps, which can be run and used to test out various flows of the SDK.
+In order to run the applications, you have to define some gradle properties in your global or local `gradle.properties`.
+Whether you are using the Consumer or B2B workbench app, you must provide all of the below values, as Gradle will attempt to build all of the apps in the project
 #### **For Consumer Applications**
 - `STYTCH_PUBLIC_TOKEN` - This token can be retrieved from your Stytch dashboard as mentioned before in this README.
 - `GOOGLE_OAUTH_CLIENT_ID` - This client ID is configured in your Google OAuth settings. You can leave it blank if you are not testing Google OneTap
+- `PASSKEYS_DOMAIN` - This is the domain that hosts your `.well-known/assetlinks.json` file. You can leave it blank if you are not testing Passkeys
 #### **For B2B Applications**
 - `STYTCH_B2B_PUBLIC_TOKEN` - This token can be retrieved from your Stytch dashboard as mentioned before in this README.
-- `STYTCH_B2B_ORG_ID` - You must create an organization in your Stytch product, and retrieve this ID
+- `STYTCH_B2B_ORG_ID` - You must create an organization in your Stytch dashboard, and retrieve this ID
 
 ### Authenticating
 

